modules/services/mako.nix

```
@@ -4,13 +4,13 @@ with lib;
 
 let
 
-  cfg = config.programs.mako;
+  cfg = config.services.mako;
 
 in {
   meta.maintainers = [ maintainers.onny ];
 
   options = {
```

Please add the following to let users know of the change.

```suggestion
  imports = [
    (mkRenamedOptionModule [ "programs" "mako" ] [ "services" "mako" ])
  ];

  options = {
```

---

Does this actually do the right thing? The program added mako to `home.packages` while the service does not and instead starts the systemd service when `enable` is set to true. This change in behavior seems like a bad thing. I assume `mkRenamedOptionModule` does not handle this by itself, does it?

---

No, it instructs users to use `services.mako` instead of `programs.mako` if they try building their configuration with `programs.mako`.